### PR TITLE
delete X-UA-Compatible

### PR DESCRIPTION
**Pull request by Otakan951**
Mastodon doesn't work on IE, but it has "X-UA-Compatible" content="IE=edge".

### CODE REVIEW

**Comment by nightpool** [comment]
@nolanlawson thoughts?

**Review by akihikodaki — APPROVED** [review]

File: spec/fixtures/requests/idn.txt [hunk 1]
```diff
@@ -11,7 +11,6 @@ Accept-Ranges: bytes
 <html>
 	<head>
 		<meta charset="UTF-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1" />
```
Comment (ykzts) [inline spec/fixtures/requests/idn.txt]: I think it is unnecessary.

Comment (yumetodo) [reply spec/fixtures/requests/idn.txt]: @ykzts Do you mean this should be replaced to `<meta http-equiv="X-UA-Compatible" content="IE=edge" />`?(Google Chrome Frame is not supported)

Comment (ykzts) [reply spec/fixtures/requests/idn.txt]: This file is used for testing only. It simulates the contents of an external web page and has nothing to do with Mastodon.

**Pull request merged**